### PR TITLE
[DISKPART] Update Romanian translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: base/system/diskpart/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,4 +1,10 @@
-/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) */
+/*
+ * PROJECT:     ReactOS Diskpart
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
```
Comment (julenuri) [inline base/system/diskpart/lang/ro-RO.rc:3]:
```suggestion
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```

**Review by julenuri — CHANGES_REQUESTED** [review]

**Comment by GeoB99** [comment]
I'm going to take a look at this and other translations of yours, stay tuned!

File: base/system/diskpart/lang/ro-RO.rc [hunk 2]
```diff
@@ -19,39 +25,39 @@ END
...
 STRINGTABLE
 BEGIN
-    IDS_DETAIL_INFO_DISK_ID "Disc ID: %08lx\n"
+    IDS_DETAIL_INFO_DISK_ID "ID-ul discului: %08lx\n"
     IDS_DETAIL_INFO_TYPE "Tip"
     IDS_DETAIL_INFO_STATUS "Stare"
     IDS_DETAIL_INFO_PATH   "Cale   : %hu\n"
     IDS_DETAIL_INFO_TARGET "Țintă  : %hu\n"
-    IDS_DETAIL_INFO_LUN_ID "LUN ID : %hu\n"
+    IDS_DETAIL_INFO_LUN_ID "ID-ul LUN-ului : %hu\n"
```
Comment (GeoB99) [inline base/system/diskpart/lang/ro-RO.rc:60]: IIRC, English specific terms or acronyms are better left as is. `ID-ul LUN-ului` would imply that LUN is a Romanian acronym which is that not the case. I would also keep `Disc ID`.

Comment (AndreiMiloiu) [reply base/system/diskpart/lang/ro-RO.rc]: OK, I changed them back.

**Review by julenuri — APPROVED** [review]

**Pull request merged**